api/Dockerfile

```diff
@@ -8,19 +8,22 @@ COPY ./ $HOME/api
 WORKDIR $HOME/api
 
 RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y nginx curl
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y git nginx curl
 
 RUN export NODEV='16.9.0' \
     && curl "https://nodejs.org/dist/v${NODEV}/node-v${NODEV}-linux-x64.tar.gz" | tar -xzv \
     && cp ./node-v${NODEV}-linux-x64/bin/node /usr/bin/ \
     && ./node-v${NODEV}-linux-x64/bin/npm install -g npm@7
 
 RUN npm install \
+    && npm uninstall bcrypt && npm install bcrypt \
```

Comment:
Could you add a comment or so saying why you needed to do this? (Even if its something like "I dont know why, but I needed to do this to make it work.") - will make it helpful in the future if someone wants to remove it / is wondering why it's there.

Reply:
> Error: /app/node_modules/bcrypt/lib/binding/bcrypt_lib.node: invalid ELF header

I was getting this error in my local docker setup, after reading I figured out it has to do something with how `bcrypt` compiles across different OS.  Here are the links that I referred to make it work [link1](https://www.richardkotze.com/top-tips/install-bcrypt-docker-image-exclude-host-node-modules) & [link2](https://medium.com/hacktive-devs/the-bcrypt-bg-on-docker-9bc36cc7f684).